.github/workflows/create_release_draft.yml

```diff
@@ -16,89 +16,75 @@ on:
       - .github/**
       - utils/**
 
+env:
+  OUT_DIR: ${{ github.workspace }}/../grass_outdir
+  GRASS: grass-${{ github.ref_name }}
+permissions:
+  contents: write
```

Comment:
```suggestion
permissions: {}
```

Clear all default permissions, and below add back what is needed, per job (also to not mistakenly have more permissions for new jobs inside the same workflow).

Reply:
Changed.